src/Strategy/NullCacheStrategy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Kevinrob\GuzzleCache\Strategy;
+
+use Kevinrob\GuzzleCache\CacheEntry;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class NullCacheStrategy implements CacheStrategyInterface
+{
+
+    /**
+     * @inheritDoc
+     */
+    public function fetch(RequestInterface $request)
+    {
+        return null;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function cache(RequestInterface $request, ResponseInterface $response)
+    {
+        return true;
```

Comment:
> However, to my eyes, nothing failed: the NullCacheStrategy accomplished its job with success (i.e. doing nothing).

Yes, you're right. I'm not familiar with this pattern, but I understand it. Thanks for your point of view!